### DISCUSSION TITLE
How to disable automatic inertion of closing braces or quotes

### DISCUSSION THREAD

**Discussion by mateuszaaa** [LazyVim/LazyVim#2008] [Q&A]
HI, i couldnt find clear information about which plugin enables automatic inertion of closing brace or quote when typing in insert mode.

Essentially when i type `"` i would like it to just insert `"` and not `""`. Same with braces when i type `(` i would not like to `)` to be auotmatically inserted.

Best Regards

**Comment by dpetka2001** [comment]
It's `mini.pairs`. You can either disable the plugin or in insert mode you can press `<C-v>` before inserting the character you want. `<C-v>` will cause the character that you press next to be inserted literally without any effect from plugins (in case you still want to keep the plugin around for other pairs). You can also disable pair autocompletion for specific pairs of characters according to filetypes. See #1931 for more info.

**Comment by ianmarcinkowski** [comment]
Commenting because this was driving me crazy!

As of 2025-09-26 the instructions for disabling this plugin can be found https://github.com/nvim-mini/mini.pairs/blob/main/doc/mini-pairs.txt

In my nvim configuration directory I have added the following to `lua/config/options.lua` to completely disable `mini.pairs`:

lua/config/options.lua:

```lua
vim.g.minipairs_disable = true
```

**Reply by jchrisjones-sap** [reply]
Thank you!
Also you can create a disabled.lua file in your plugins folder and add what you need:
```lua
return {
  { "nvim-mini/mini.pairs", enabled = false },
  { "altermo/ultimate-autopair.nvim", enabled = false },
}
```

Complements of this thread: https://www.reddit.com/r/neovim/comments/18feq5j/lazyvim_how_to_turn_default_plugins_off/
